fast_loop.h

```diff
@@ -244,6 +244,7 @@ class FastLoop {
       ia_bias_ = param_.ia_bias;
       ib_bias_ = param_.ib_bias;
       ic_bias_ = param_.ic_bias;
+      max_i_bias_ = param_.max_i_bias == 0 ? 10 : param_.max_i_bias;
```

Comment:
do you really want a direct float comparison? might suggest checking it's < epsilon

Reply:
Yeah I like direct float comparisons. I use them everywhere. Especially to 0 I think it's safe.

Reply:
yeah I think for zero it is, so I guess this is fine, but I don't think it's great if we are doing anything arithmetic.